bentoml/yatai/web/src/index.ts

```diff
@@ -0,0 +1,11 @@
+import { getExpressApp } from './server'
+
+// 0 is node, 1 is index.ts
+const args = process.argv.slice(2);
+// going to assume channel address at 0 and port at 1 for now
+const grpcServerAddress = args[0] || 'localhost:50051';
```

Comment:
let's make both of them required positional arg since it's only supposed to be called by BentoML code
```
const args = process.argv.slice(2);

if args.lenth !== 2:
   print_warning_and_show_usage
   exit

const grpcServerAddress = args[0]
const port = args[1]
```